modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,692 @@
+
+**Advanced Code Editor** 3.1.0 includes the following improvement.
+
+
+==== New option to display the Advanced Code Editor inline instead of in a dialog
+
+In {productname} 6.3, the Advanced Code Editor plugin includes a new option, `advcode_inline`, that allows users to open the Advanced Code Editor within TinyMCE’s existing editor space instead of being displayed in a separate dialog box.
+
+The basic setup for this new option as follows.
+
+[source,js]
+----
+tinymce.init({
+  selector: "textarea", // change this value according to your HTML
+  plugins: "advcode",
+  advcode_inline: true, // To use this option, set this value to "true".
+  toolbar: "code",
+});
+----
+
+=== Checklist 2.0.2
+
+The {productname} 6.3 release includes an accompanying release of the **Checklist** premium plugin.
+
+**Checklist** 2.0.2 includes one fix:
+
+==== Checklist toggling did not preserve selection when used on an existing list
+
+In previous {productname} versions, when selecting content over either an ordered or unordered list, then switching the list to a checklist, the user’s content selection was not preserved.
+
+In {productname} 6.3, the user selection is now preserved when toggling a list to a checklist.
+
+For information on the **Checklist** plugin, see xref:checklist.adoc[Checklist].
+
+=== Comments 3.2.0
+
+The {productname} 6.3 release includes an accompanying release of the **Comments** premium plugin.
+
+**Comments** 3.2.0 includes one fix and one change:
+
+==== A comment could be inserted when in an empty editor or block
+
+In previous {productname} versions a bug existed that allowed comments to be placed into an empty editor or block.
+
+With this update, the comments button, menu-item, comment sidebar and shortcut keys are now disabled when the editor contains no content.
+
+==== The selection now expands to encompass the word containing the insertion point when a comment is added
+
+{productname} 6.3 adds xref:additions[two new API functions] — `tinymce.selection` and `tinymce.dom.RangeUtils`.
+
+With these two functions available, if there is no selected text when a new comment is added, {productname} automatically extends the selection to encompass the word containing the insertion point and links the newly added comment to this automatically selected word.
+
+For information on the **Comments** plugin, see: xref:introduction-to-tiny-comments.adoc[Introduction to Tiny Comments].
+
+
+=== Merge Tags 1.1.0
+
+The {productname} 6.3 release includes an accompanying release of the **Merge Tags** premium plugin.
+
+**Merge Tags** 1.1.0 includes one addition, one fix, and one change:
+
+==== New mceMergeTag command to wrap selected content in a merge tag
+
+A new command, `mceMergeTag` has been added which can convert any string that include Merge Tag delimiter characters into a merge tag. This new command is available to use when defining xref:content-behavior-options.adoc#text_patterns[Text Patterns].
```

Comment:
> A new command, `mceMergeTag` has been added which can convert any string that include Merge Tag delimiter characters into a merge tag.

That is not correct. The command can be applied to any selected content that does not necessarily contain merge tag delimited characters but instead wraps selected content in the prefix and suffix and then makes the result noneditable.

Reply:
to quote from [this commit](https://github.com/tinymce/tinymce-docs/pull/2574/commits/5ecada0a466638b65027a8fff094fd0889ed3e9e):

> Regarding mceMergeTag introductory material: further corrections to further corrections.